include/boost/spirit/home/x3/support/context.hpp

```diff
@@ -7,7 +7,21 @@
 =============================================================================*/
 #if !defined(BOOST_SPIRIT_X3_CONTEXT_JAN_4_2012_1215PM)
 #define BOOST_SPIRIT_X3_CONTEXT_JAN_4_2012_1215PM
-
+#if 0
+  #pragma message "deprecated.  May cause infinite template recursion when using x3::skip(skipper)[some recursive rule]."
+  //For example, see 
+  /*
+    https://stackoverflow.com/questions/45282293/spirit-not-able-to-use-x3skipskippersome-recursive-rule-in-its-rule-defin
+   */
+  //Mailing list discussion includes:
+  /*
+    https://sourceforge.net/p/spirit/mailman/message/35965692/
+   */
+  //Solution in PR237
+  /*
+    https://github.com/boostorg/spirit/pull/237
+   */
+#endif
```

Comment:
Why it is here if it is disabled?

Reply:
I simply overlooked it.  Thanks for finding it and I'll remove.